e2e/app_test.go

```diff
@@ -69,7 +69,53 @@ func TestAppRetryCount(t *testing.T) {
 	}
 }
 
-// Checks if the stats are written at the end of each log level output.
+// Checks if --stat flag does not print when used with help & version commands
+func TestAppDashStatUnnecessaryPrints(t *testing.T) {
```

Comment:
This test looks the same as `TestAppDashStat`. We should merge them.

Reply:
Just merged `TestAppDashStatUnnecessaryPrints` and  `TestAppDashStat` into one under the name `TestAppDashStat`.